modules/bound-sa-tokens-configuring.adoc

```diff
@@ -17,6 +17,21 @@ You can configure pods to request bound service account tokens by using volume p
 . Optionally, set the service account issuer.
 +
 This step is typically not required if the bound tokens are used only within the cluster.
++
+[WARNING]
+====
+If you update the `serviceAccountIssuer` field and there are bound tokens already in use, all bound tokens with the previous issuer value will be invalidated. Unless the holder of a bound token has explicit support for a change in issuer, they will not request a new bound token until pods have been restarted.
+
+If necessary, you can use the following command to manually restart all pods in the cluster. Be aware that running this command causes a service interruption, because it deletes every running pod in every namespace. These pods will automatically restart after they are deleted.
```

Comment:
Note for reviewer: everything starting with "you can use " to the end of the warning is pulled from a note in this section: https://docs.openshift.com/container-platform/4.7/security/certificates/service-serving-certificate.html#understanding-service-serving_service-serving-certificate

Reply:
The new note LGTM!